File: go/wam/4814.go
const WamWebDbLoadFromVersionFailureNonAnonymous = 4814 // channel: regular

type WebDbLoadFromVersionFailureNonAnonymousEvent struct {
	WebDbLoader WEBDBLOADERTYPE `wam:"web_db_loader"` // field 1
	WebDbName WEBDBNAMETYPE `wam:"web_db_name"` // field 2
}

File: ts/wam/4814.ts
export const WamWebDbLoadFromVersionFailureNonAnonymous = 4814 as const

export interface WebDbLoadFromVersionFailureNonAnonymousEvent {
  /** field 1, wire `web_db_loader` */
  webDbLoader?: WEBDBLOADERTYPE
  /** field 2, wire `web_db_name` */
  webDbName?: WEBDBNAMETYPE
}